docs/sphinx/theory/problem_formulation.rst

```diff
@@ -12,7 +12,14 @@ The current version of the view factor model only addresses PV rows that are mad
    :width: 80%
    :align: center
 
-By making some assumptions, it is possible to represent the calculation of irradiance terms on each surface with a linear system. The dimension of this system changes depending on the number of surfaces considered. But we can formulate it for the general case of ``n`` surfaces.
+
+The mathematical model used in pvfactors simulations is different depending on the simulation type that is run. In "full simulations", all of the reflections between the modeled surfaces are taken into account in the calculations. In "fast simulations", assumptions are made on the reflected irradiance from the environment surrounding the surfaces of interest.
```

Comment:
Maybe we could add a CircleCI artifact so that we can check doc rendering online?

Reply:
hey thanks @cedricleroy that was a great advice, I just added this to circleci and you can check out the docs in the circleci artefacts now, hopefully that will help with the review